sdk/netapp/Azure.ResourceManager.NetApp/api/Azure.ResourceManager.NetApp.netstandard2.0.cs

```diff
@@ -709,6 +714,16 @@ public partial class CapacityPoolPatch : Azure.ResourceManager.Models.TrackedRes
         public static bool operator !=(Azure.ResourceManager.NetApp.Models.EnableNetAppSubvolume left, Azure.ResourceManager.NetApp.Models.EnableNetAppSubvolume right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class GetGroupIdListForLdapUserContent
+    {
+        public GetGroupIdListForLdapUserContent(string username) { }
+        public string Username { get { throw null; } }
+    }
+    public partial class GetGroupIdListForLdapUserResponse
```

Comment:
shall we rename this to `GetGroupIdListForLdapUserResult`? Usually we should avoid `Response` suffix